dapp/tests/auctions/auctions.spec.ts

```diff
@@ -1,11 +1,70 @@
 // Copyright (c) 2025 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { expect } from '@playwright/test';
+import { expect, test } from '../helpers/fixtures';
+import {
+    connectWallet,
+    createWallet,
+    generateRandomName,
+    requestFaucetTokens,
+    transactionToCreateAnAuction,
+} from '../utils';
 
-import { test } from '../helpers/fixtures';
+test.describe.serial('Auction Bid Flow', () => {
+    test.beforeAll(async ({ appPage, context, extensionPage, extensionName, sharedState }) => {
+        const { address, mnemonic } = await createWallet(extensionPage);
 
-test('Auction flow works', async ({ appPage }) => {
-    await expect(appPage).toHaveTitle(/IOTA Names/);
-    await expect(appPage.getByText('Your On-Chain Name')).toBeVisible();
+        await appPage.bringToFront();
+        await connectWallet(appPage, context, extensionName);
+
+        await requestFaucetTokens(address);
+        sharedState.wallet.address = address;
+        sharedState.wallet.mnemonic = mnemonic;
+
+        sharedState.testAuctionName = generateRandomName();
+    });
+
+    test('create bid on existing auction', async ({ sharedState, appPage: page }) => {
+        await transactionToCreateAnAuction({ sharedState, useNewWallet: true });
+
+        await page.goto('/auctions');
```

Comment:
so that it only shows auctions related to this specific test
```suggestion
        await page.goto(`/auctions?page=1&search=${testAuctionName}`);
```